docs/add-ins/addin-tutorial.md

```diff
@@ -702,31 +806,29 @@ function buildBodyContent(gist, callback) {
 }
 ```
 
-If the gist contains HTML, then it will be inserted as-is into the body. If the gist is Markdown, the add-in will convert using Showdown and insert the resulting HTML. For anything else, the add-in will insert it as a code snippet.
+### Test the button
 
-#### Test the button
+Save all of your changes and run `npm start` from the command prompt, if the server isn't already running. Then complete the following steps to test the **Insert default gist** button. 
 
-The **Insert Default Gist** button should now work. Save all of your changes and run `npm start` if the server isn't already running. Open Outlook and compose a new message. When you click on the **Insert Default Gist** button, you should be prompted to configure the add-in.
+1. Open Outlook and compose a new message. 
 
-![A screenshot of the add-in's prompt to configure](images/addin-tutorial/addin-prompt-configure.PNG)
+1. In the compose message window, select the **Insert default gist** button. You should be prompted to configure the add-in.
 
-Enter your GitHub username. Press **Tab** to invoke the `change` event, which should load your list of gists. Select a gist to be the default, and click **Done**.
+    ![A screenshot of the add-in's prompt to configure](images/addin-tutorial/addin-prompt-configure.PNG)
 
-> [!NOTE]
-> If you don't have any gists on GitHub, go create some!
+1. In the settings dialog, enter your GitHub username and press **Tab** to invoke the `change` event, which should load your list of gists. Select a gist to be the default, and select **Done**.
```

Comment:
Tab or click away. Not sure if that's worth mentioning.

Reply:
I'll reword this.